docs/getting-started/index.md

```diff
@@ -0,0 +1,134 @@
+
+Below outlines the main steps for doing your first deployment and some of the terms you will need to know.
+
+### 1. Hosting the Octopus Deploy Server
+
+You can install your own [self-hosted](#self-hosted-octopus) instance of the Octopus Deploy Server or use [Octopus Cloud](#octopus-cloud).
+
+#### Self-Hosted Octopus {#self-hosted-octopus}
+
+Installing the self-hosted [Octopus Deploy server](/docs/installation/index.md) sets up the [Octopus Web Portal](/docs/getting-started/index.md#the-octopus-web-portal) and the [Octopus REST API](/docs/octopus-rest-api/index.md). The [installation documentation](/docs/installation/index.md) provides instructions for downloading, installing, and configuring your Octopus Deploy server.
+
+#### Octopus Cloud {#octopus-cloud}
+
+Octopus Cloud is the hosted version of Octopus Deploy. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, upgrading the service, and maintaining and monitoring the underlying systems.
+
+Learn more about [Octopus Cloud](/docs/octopus-cloud/index.md).
+
+#### The Octopus Web Portal
+
+Whether you're self-hosting the Octopus server, or using Octopus Cloud, the Octopus Web Portal is where you'll manage your infrastructure, projects, access the built-in repository, grant your team access to projects, and create your automated deployments.
+
+### 2. Set up deployment environments
+
+Octopus will need to know where you are going to deploy to. You can deploy to Windows servers, Linux servers, Microsoft Azure, an Offline Package Drop, Cloud Regions, or Kubernetes. These are known as your deployment targets, and they are organized into environments so you can promote your software through your deployment pipeline, for instance, from Development to Testing and finally into Production.
+
+#### Deployment Targets
+
+Deployment targets represent the servers, machines and cloud services where your software and services will be deployed.
+
+#### Environments
+
+Octopus organizes your deployment targets into groups called environments so you can promote your software through your deployment pipeline, for instance, from Development to Test and finally into Production.
+
+#### Target Roles
+
+Target roles allow you to “tag” deployment targets with a specific keyword which can be used in your deployments.
+
+#### Tentacle Agent
+
+When you deploy software to Windows servers, you need to install Tentacle Agent, a lightweight agent service, on your Windows servers so they can communicate with the Octopus server in either a listening or a polling mode.
```

Comment:
With the Linux Tentacle, this statement is no longer accurate.  Perhaps, "When you deploy to servers, running either Windows, Linux or Mac, you need to install the Tentacle Agent...."

Also, I'm wondering if we should include a blurb about not needing a tentacle if you are deploying to Azure Web Apps, K8s clusters, Service Fabric, etc